### PR TITLE
Prior tests

### PR DESCRIPTION
**Pull request by yannikschaelte**
* fix assert_prior_type_is_valid (did not handle empty entries well)
* add assert_prior_parameters_are_valid
* add tests for both

### CODE REVIEW

File: tests/test_lint.py [hunk 1]
```diff
@@ -258,13 +258,30 @@ def test_check_parameter_bounds():
...
+         OBJECTIVE_PRIOR_TYPE: [NORMAL, LOG_NORMAL, '']}))
     lint.assert_parameter_prior_type_is_valid(pd.DataFrame())
 
     with pytest.raises(AssertionError):
         lint.assert_parameter_prior_type_is_valid(pd.DataFrame(
-            {INITIALIZATION_PRIOR_TYPE: ['normal', '']}))
+            {INITIALIZATION_PRIOR_TYPE: ['normel']}))
+
+
+def test_assert_parameter_prior_parameters_are_valid():
```
Comment (dweindl) [inline tests/test_lint.py]: Codacy Issue found: [Missing function or method docstring](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4984274)

File: petab/lint.py [hunk 2]
```diff
@@ -404,15 +404,62 @@ def assert_parameter_prior_type_is_valid(
...
+                # empty parameters fine
+                continue
+            # parse parameters
+            try:
+                pars = tuple([float(val) for val in pars_str.split(';')])
+            except ValueError:
+                raise AssertionError(
+                    f"Could not parse prior parameters '{pars}'.")
+            # all distributions take 2 parameters
+            if 2 != len(pars):
```
Comment (dweindl) [inline petab/lint.py]: Codacy Issue found: [Comparison should be len(pars) != 2](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4984274)

Comment (yannikschaelte) [reply petab/lint.py]: @dweindl no your bot is wrong here. it is common to use this form to prevent unintended assignments instead of checks.

Comment (dweindl) [reply petab/lint.py]: I am with the bot here. More readable. :)

Your argument doesn't count in python.

```
    if a = 2: print(test)
         ^
SyntaxError: invalid syntax
```

**Pull request merged**